todo_app/app.py

```diff
@@ -1,11 +1,27 @@
-from flask import Flask
+from flask import Flask, render_template
+from flask import request
+from todo_app.data.session_items import get_items
+from todo_app.data.session_items import add_item
```

Comment:
It's a matter of taste rather than functionality, but as these imports grow you may well find it's nicer to collect similar imports from the same module (like you have `Flask` and `render_template`):
```python
from todo_app.data.session_items import get_items, add_item
```